src/fields/BooleanField.cpp

```diff
@@ -1,17 +1,24 @@
-#include "BooleanField.h"
-
-BooleanField::BooleanField(string fieldName, bool defaultValue, bool isPrimaryKey, bool isUnique, bool isNullable):
-    Field(fieldName, BOOLEAN, isPrimaryKey, isUnique, isNullable), _fieldValue(defaultValue) {}
-
-void BooleanField::setValue(bool value) {
-    setNull(false);
-    _fieldValue = value;
-}
-
-bool BooleanField::getValue() {
-    return _fieldValue;
-}
-
-void BooleanField::setParameter(int parameterNumber, DBConnection connection) {
-    connection.setInt(parameterNumber, getValue());
-}
+
+#include "BooleanField.h"
+
+BooleanField::BooleanField(string fieldName, bool defaultValue, bool isPrimaryKey, bool isUnique, bool isNullable):
+    Field(fieldName, BOOLEAN, isPrimaryKey, isUnique, isNullable), _fieldValue(defaultValue), _defaultValue(defaultValue) {}
```

Comment:
I am actually also starting to wonder is `isAutoFilled` is necessary. I must check my old scripts to see what I used to use it for...

For now, I will add it.